src/pyspiffe/workloadapi/default_workload_api_client.py

```diff
@@ -1,12 +1,22 @@
-from typing import Optional, Set
+from typing import Optional, Set, Tuple
+
+import grpc
+
 from pyspiffe.bundle.x509_bundle.x509_bundle_set import X509BundleSet
 from pyspiffe.bundle.jwt_bundle.jwt_bundle_set import JwtBundleSet
 from pyspiffe.config import ConfigSetter
+from pyspiffe.proto.spiffe import (
+    workload_pb2_grpc,
+    workload_pb2,
+)
+from pyspiffe.workloadapi.grpc import header_manipulator_client_interceptor
 from pyspiffe.svid.x509_svid import X509Svid
 from pyspiffe.svid.jwt_svid import JwtSvid
 
 from pyspiffe.workloadapi.workload_api_client import WorkloadApiClient
 
+_WORKLOAD_API_HEADER = 'workload.spiffe.io'
```

Comment:
Do you think this should be in the abstract class, or here?

Reply:
I'll move it to the abstract class, it makes more sense. Thx.